### ISSUE TITLE
FATAL: Caught exception: File size is not a multiple of recordSize

### ISSUE THREAD

**Issue by freerko** [cculianu/Fulcrum#122]
Hi,

looks like my VM got rebootet and afterwards Fulcrum doesn't start anymore with the error `FATAL: Caught exception: File size is not a multiple of recordSize`. Do I have to delete the db folder and sync again from scratch or is there anything else I can try to fix this?

    Jun 15 07:17:00 server Fulcrum[82589]: Loaded SSL certificate: Internet Widgits Pty Ltd  expires: Tue March 12 2026 12:42:59
    Jun 15 07:17:00 server Fulcrum[82589]: Loaded key type: private algorithm: RSA
    Jun 15 07:17:00 server Fulcrum[82589]: Enabled JSON parser: simdjson
    Jun 15 07:17:00 server Fulcrum[82589]: simdjson implementations:
    Jun 15 07:17:00 server Fulcrum[82589]:     haswell: Intel/AMD AVX2  [supported]
    Jun 15 07:17:00 server Fulcrum[82589]:     westmere: Intel/AMD SSE4.2  [supported]
    Jun 15 07:17:00 server Fulcrum[82589]:     fallback: Generic fallback implementation  [supported]
    Jun 15 07:17:00 server Fulcrum[82589]: active implementation: haswell
    Jun 15 07:17:00 server Fulcrum[82589]: jemalloc: version 5.2.1-0-gea6b3e9
    Jun 15 07:17:00 server Fulcrum[82589]: Qt: version 5.15.2
    Jun 15 07:17:00 server Fulcrum[82589]: rocksdb: version 6.14.6-ed43161
    Jun 15 07:17:00 server Fulcrum[82589]: simdjson: version 0.6.0
    Jun 15 07:17:00 server Fulcrum[82589]: ssl: OpenSSL 1.1.1f  31 Mar 2020
    Jun 15 07:17:00 server Fulcrum[82589]: zmq: libzmq version: 4.3.3, cppzmq version: 4.7.1
    Jun 15 07:17:00 server Fulcrum[82589]: Fulcrum 1.7.0 (Release 4ee413a) - Wed Jun 15, 2022 07:17:00.367 UTC - starting up ...
    Jun 15 07:17:00 server Fulcrum[82589]: Max open files: 1048576 (increased from default: 1024)
    Jun 15 07:17:00 server Fulcrum[82589]: Loading database ...
    Jun 15 07:17:05 server Fulcrum[82589]: DB memory: 512.00 MiB
    Jun 15 07:17:05 server Fulcrum[82589]: Coin: BTC
    Jun 15 07:17:05 server Fulcrum[82589]: Chain: main
    Jun 15 07:17:05 server Fulcrum[82589]: FATAL: Caught exception: File size is not a multiple of recordSize
    Jun 15 07:17:05 server Fulcrum[82589]: Stopping Controller ...
    Jun 15 07:17:05 server Fulcrum[82589]: Closing storage ...
    Jun 15 07:17:05 server Fulcrum[82589]: Shutdown complete

**Comment by cculianu** [comment]
Yeah the db is corrupted.  You must now delete the data dir and resynch.  

If you have extra diskspace leftover, after a resynch, you can stop fulcrum then take a backup of the datadir.  This way you have at least a partially synched Fulcrum datadir backup you can restore from...
